addon/components/lazy-options/template.hbs

```diff
@@ -0,0 +1,43 @@
+<!--  https://github.com/cibernox/ember-power-select/blob/master/addon/templates/components/power-select/options.hbs -->
+{{#if select.loading}}
+    {{#if loadingMessage}}
+        <li class="ember-power-select-option ember-power-select-option--loading-message" role="option">{{loadingMessage}}</li>
+    {{/if}}
+{{/if}}
+{{#each options as |opt index|}}
+    {{#if (ember-power-select-is-group opt)}}
+        {{#component groupComponent
+                     group=(readonly opt)
+                     select=(readonly select)
+                     extra=(readonly extra)
+        }}
+            {{#component optionsComponent
+                         options=(readonly opt.options)
+                         select=(readonly select)
+                         groupIndex=(concat groupIndex index ".")
+                         optionsComponent=(readonly optionsComponent)
+                         groupComponent=(readonly groupComponent)
+                         extra=(readonly extra)
+                         role="group"
+                         class="ember-power-select-options" as |option|}}
+                {{yield option select}}
+            {{/component}}
+        {{/component}}
+    {{else}}
+        <li class="ember-power-select-option"
+            aria-selected="{{ember-power-select-is-selected opt select.selected}}"
+            aria-disabled={{ember-power-select-true-string-if-present opt.disabled}}
+                aria-current="{{eq opt select.highlighted}}"
+            data-option-index="{{groupIndex}}{{index}}"
+            role="option">
+            {{yield opt select}}
+        </li>
+    {{/if}}
+{{/each}}
+{{#if (or isLoading)}}
```

Comment:
It looks like there's only one condition in this if.  Was there a reason to have an `or`?

Reply:
It seems that I removed one of the variables and forgot to get rid of the 'or'...